ngrok/src/internals/proto.rs

```diff
@@ -571,12 +682,10 @@ pub struct TlsEndpoint {
 }
 
 #[derive(Serialize, Deserialize, Debug, Clone, Default)]
-#[serde(rename_all = "PascalCase")]
 pub struct TlsTermination {
     #[serde(with = "base64bytes")]
     pub cert: Vec<u8>,
-    #[serde(with = "base64bytes")]
-    pub key: Vec<u8>,
+    pub key: SecretBytes,
     #[serde(with = "base64bytes")]
     pub sealed_key: Vec<u8>,
```

Comment:
since sealed_key is unused does this need a skip_serializing_if default?

Reply:
Whoops, looks like I just missed this struct entirely.